src/components/landscape-sidebar/geo-description-widget/geo-description-widget-component.jsx

```diff
@@ -40,4 +48,4 @@ const GeoDescriptionWidget = ({ data, loading, error, view }) => {
   );
 };
 
-export default GeoDescriptionWidget;
+export default connect(null, actions)(GeoDescriptionWidget);
```

Comment:
Please do not `connect` the `component`. Following the `container`/`component` pattern we usually make the `component` as 'dumb' as possible, just passing the needed props from the `container` to the `component` and leaving the latest just for rendering purposes.